src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -15,7 +15,7 @@ public partial class PickerHandler : ViewHandler<IPicker, MauiPicker>
 		AppCompatAlertDialog? _dialog;
 
 		protected override MauiPicker CreatePlatformView() =>
-			new MauiPicker(Context);
+			new MauiPicker(Context){ KeyListener = null };
```

Comment:
Could we apply the property inside the `MauiPicker` class?

Reply:
@jsuarezruiz , I have modified the code changes. Please let me know if you have any concerns.